plugins/dev-create/bin/app.js

```diff
@@ -0,0 +1,64 @@
+const chalk = require('chalk');
+const fs = require('fs-extra');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const {createDir} = require('./common');
+
+exports.createApp = function(name, options) {
+  if (options.typescript) {
+    console.error('Not yet implemented');
+    process.exit(1);
+  }
+
+  const root = process.cwd();
+  const dir = options.dir ?? name;
+  const appPath = path.join(root, dir);
+
+  // Create the new directory for the app if needed.
+  createDir(appPath);
+  console.log(
+      `Creating a new Blockly application called ${name} in ${appPath}`);
+
+  // Copy over files from sample-app directory.
+  const sampleDir = '../../../examples/sample-app';
+  fs.copySync(path.resolve(__dirname, sampleDir), appPath, {
+    filter(src) {
+      const excludes = [
+        'node_modules', 'dist', 'package-lock.json', 'package.json'];
+      for (const file of excludes) {
+        if (src.indexOf(file) > -1) return false;
+      }
+      return true;
+    },
+  });
+
+  // Create package.json file for app.
+  const templateJson = require(path.join(sampleDir, 'package.json'));
+  const packageJson = {
+    ...templateJson,
+    name: name,
+    author: options.author || '',
+  };
+
+  // Write the package.json to the new package.
+  fs.writeFileSync(
+      path.join(appPath, 'package.json'),
+      JSON.stringify(packageJson, null, 2));
+
+  // Run `npm install` to get things ready for user.
+  if (!options.skipInstall) {
+    console.log('Installing packages. This might take a couple of minutes.');
+    execSync(`cd ${dir} && npm install`, {stdio: [0, 1, 2]});
```

Comment:
I think `'inherit'` is clearer:
```suggestion
    execSync(`cd ${dir} && npm install`, {stdio: 'inherit'});
```